statesync/syncer.go

```diff
@@ -256,7 +260,7 @@ func (s *syncer) Sync(snapshot *snapshot, chunks *chunkQueue) (sm.State, *types.
 		go s.fetchChunks(ctx, snapshot, chunks)
 	}
 
-	pctx, pcancel := context.WithTimeout(context.Background(), 10*time.Second)
+	pctx, pcancel := context.WithTimeout(context.Background(), 15*time.Second)
```

Comment:
could we add a comment for why this time was chosen

Reply:
Well the motivation for the change was that I was seeing deadline errors with the e2e tests but apart from that the decision is quite arbitrary

Reply:
does this timeout effect live networks?

Reply:
what exactly do you mean by that?

Reply:
Do we see this timeout accidentally being exceeded on main nets?

Reply:
I wouldn't know